pkg/monitor/cluster/clusterwideproxystatus.go

```diff
@@ -44,7 +44,14 @@ func (mon *Monitor) emitCWPStatus(ctx context.Context) error {
 		})
 	} else {
 		// Create the noProxy map for efficient lookups
-		no_proxy_list := strings.Split(proxyConfig.Spec.NoProxy, ",")
+		no_proxy_list := strings.Split(proxyConfig.Status.NoProxy, ",")
```

Comment:
I suggest we sanitize NoProxy first before using it. Remove trailing spaces and trailing ",". If there's a typo such as a trailing "," or " ", assuming this is not invalidated yet somewhere, then this will cause the NoProxy to return a non-empty string value, and it may also cause the no_proxy_list array to have an extra item after splitting.

Reply:
Good point @pepedocs : Will update it.

Reply:
@pepedocs : a blank space or typo error like ,, or , ,  will result in error from the network operator stating invalid proxy.
ex: `"The configuration is invalid for proxy 'cluster' (invalid noProxy: ). Use 'oc edit proxy.config.openshift.io cluster' to fix."`

So from the monitor perspective, I dont think we can do anything from here, but for trailing and leading space issues with the noProxy item is fixed by : strings.TrimSpace(proxy).

Reply:
@pepedocs , Could you please review the change and let me know your thoughts.